.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
 ### This PR: 
-* Describe all the points covered in the pull request (changes you might have made)
+ <!--  Describe all the points covered in the pull request (changes you might have made) -->
 
 ### Self-Review Checklist
-*(put x in between the brackets to mark them completed [x])*
+ <!-- (put x in between the brackets to mark them completed [x]) -->
```

Comment:
Did you see this done somewhere else? Seems like a good idea.

Reply:
I have seen comments in other pull request templates. Most recently, I have seen it in the pull request template of Chakra UI. [link](https://raw.githubusercontent.com/akshitarora921/chakra-ui/main/.github/pull_request_template.md)

Reply:
Nice. Yeah, I just saw this in another repo as well. Seems super common. Merging ✨